packages/block-library/src/image/edit.js

```diff
@@ -111,6 +111,7 @@ export function ImageEdit( {
 		width,
 		height,
 		sizeSlug,
+		enableLightbox = true,
```

Comment:
Not 100% sure but, shouldn't we define the [default value](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/#default-value) in the `block.json` instead of here? Something like:

```json
"enableLightbox": {
    "type": "boolean",
    "default": true
}
```